client/src/pages/admin/merge/MergePeople.tsx

```diff
@@ -97,8 +99,16 @@ const MergePeople = ({ pageDispatchers }: MergePeopleProps) => {
   const person2 = mergeState[MERGE_SIDES.RIGHT]
   const mergedPerson = mergeState.merged
 
+  useEffect(() => {
+    setIsDirty(false)
+  }, [person1, person2])
+  useEffect(() => {
+    setIsDirty(!!mergedPerson && Object.keys(mergedPerson).length > 1)
```

Comment:
Why is this different from the other three (that don't have the check on `Object.keys(…).length > 1`)?

Reply:
It may have been due to the merge bug fixed (on **main**) in https://github.com/NCI-Agency/anet/commit/9173c4ae9e1e59c234a5638f6e5ec6811109e6dc